popup/popup.js

```diff
@@ -49,28 +56,68 @@ function listenForClicks() {
     });
 }
 
-async function darkModeSwitch(){
-    if(window.darkMode){
-        await browser.tabs.removeCSS({file:"./../dark-mode/dark-mode.css"})
-    }else{
-        await browser.tabs.insertCSS({file:"./../dark-mode/dark-mode.css"})

```

Comment:
Not it's better to add the dark mode  CSS class this way through a message from the content script to the  background script?

Reply:
I've implemented it by sending a message from the background script (`popup.js`) to the content script (`storageLoader.js`).

Reply:
So what the function "addDarkMode" is for?

Reply:
This function is on the `storage.js` file and this function is the one which is actually injects the CSS to the page, I know it's a bit confusing but all of the functions in the `popup.js` are just "listeners" which call the content script to do the actual job `storage.js`

Reply:
Yeah now I understand

> I've implemented it by sending a message from the background script (`popup.js`) to the content script (`storageLoader.js`).

Yeah, now I understand what do you mean.